API.md

```diff
@@ -462,6 +463,23 @@ frisby.create('Ensure this is *actually* a real teapot, not some imposter coffee
 .toss()
 ```
 
+### expectBodyLength(length, operator)
+
+Tests that the HTTP response body has a [lengthOf](http://chaijs.com/api/bdd/#lengthOf) the provided length and it's operator (so '<','>','<=','>=','='). Used for testing all content types.
+
+| Parameter | Description | Required |
+| --------- | ----------- | -------- |
+| length | Integer that the body will be tested against | Yes |
+| operator | String of the operator that will be used to compare the length of the body | Yes |
+
+```javascript
+frisby.create('Ensure this is *actually* a real teapot, not some imposter coffee pot')
+  .get('http://httpbin.org/status/418')
+    .expectStatus(418)
+    .expectBodyLength(418,'>')
```

Comment:
If the operator is required, would it be a little clearer to put it before the number? Then this reads like an expression: "expect body length greater than 418".